onnx2torch/node_converters/cumsum.py

```diff
@@ -0,0 +1,84 @@
+from copy import deepcopy
+
+import torch
+from torch import nn
+
+from onnx2torch.node_converters.registry import add_converter
+from onnx2torch.onnx_graph import OnnxGraph
+from onnx2torch.onnx_node import OnnxNode
+from onnx2torch.utils.common import OnnxToTorchModule
+from onnx2torch.utils.common import OperationConverterResult
+from onnx2torch.utils.common import onnx_mapping_from_node
+
+
+def _arbitrary_dim_shift_and_insert_zero(
+        input_tensor: torch.Tensor,
+        insert_dim: int,
+) -> torch.Tensor:
+    t_shape = input_tensor.shape
+    insert_dim = torch.arange(start=0, end=len(t_shape), dtype=torch.int64)[insert_dim]
```

Comment:
Why do we need only positive insert_dim. Something of index_select, scatter or transpose doesnt work with negative dims?

Reply:
done,